app/models/event.rb

```diff
@@ -35,7 +40,11 @@ class Event < ActiveRecord::Base
   scope :stream, -> do
     between(Time.zone.now.beginning_of_day, 6.days.from_now.end_of_day).by_start.includes(:event_signup)
   end
+  scope :starts_within, ->(time) { where('starts_at BETWEEN :first AND :second',
```

Comment:
Maybe we should change this to
```scope :starts_within, lambda do |time|
  where(.....)
end```

which seems to be the `correct` way of doing this now.